templates/hw/task1.html

```diff
@@ -62,7 +62,7 @@ <h3>
         </p>
 
         <ol>
-            <li>average
+            <li><Code>average</Code>
```

Comment:
Are these supposed to be capitalized? Not sure if this indicates some sort of custom element, but the native HTML code element is <code>.

Reply:
Oh yea you're right. I initially had it lower case but when I had to redo the hws I guess I forgot

Reply:
Fixed, thanks for the catch!